plugin/pkg/scheduler/algorithm/predicates/predicates_test.go

```diff
@@ -1823,6 +1836,20 @@ func TestEBSVolumeCountConflicts(t *testing.T) {
 			fits:         true,
 			test:         "pod with missing PV is counted towards the PV limit",
 		},
+		{
+			newPod:       ebsPVCPod,
+			existingPods: []*v1.Pod{oneVolPod, unboundPVCPod},
+			maxVols:      2,
+			fits:         false,
+			test:         "pod with unbound PVC is counted towards the PV limit",
+		},
+		{
+			newPod:       ebsPVCPod,
+			existingPods: []*v1.Pod{oneVolPod, unboundPVCPod},
+			maxVols:      3,
+			fits:         true,
+			test:         "pod with unbound PVC is counted towards the PV limit",
```

Comment:
what about a test case if pvc doesn't exist?

Reply:
there already is a test for that, look for "pod with missing PVC is counted towards the PV limit" above.

Reply:
How about multiple PV objects not existing in the same pod and different pods?

Reply:
Added a test with two pods using the same deleted PV (=counted once) and two pods using a different deleted PV (=counted twice).